BricksAndHearts.UnitTests/ServiceTests/TestDatabaseFixture.cs

```diff
@@ -28,7 +28,9 @@ public TestDatabaseFixture()
                     CreateApprovedLandlord(), // landlordId = 1
                     CreateUnapprovedLandlord(), // landlordId = 2
                     CreateLandlordWithLink(), // landlordId = 3
-                    CreateUnlinkedLandlordWithLink() // landlordId = 4
+                    CreateUnlinkedLandlordWithLink(), // landlordId = 4
+                    CreateLandlordWithMembershipId(420), // landlordId = 5

```

Comment:
are 420 and 421 magic numbers here, do they mean anything?

Reply:
These were completely random I think. It was a long time ago but I vaguely remember some of the lower numbers were being used so I picked something a bit higher to not conflict with anything? If not though can change them to something more logical :)